README.md

```
@@ -58,98 +128,290 @@ Description: Implementation of OpenID for Verifiable Presentations - in Kotlin
 
+## Installation
 
-    Lib->>VP: HTTP POST Request with:<br/>1. VP Token<br/>2. Presentation Submission<br/>3. State
+The OpenID4VP SDK can be integrated into your Android application using Gradle, either through Android Studio or by manually updating your `build.gradle.kts`.
+
+### Option 1: Add the Package Using Android Studio
+
+1. Open your project in Android Studio.
+2. Navigate to **File > Add Package Dependencies...**.
+3. Enter the repository URL:
+   ```text
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)